interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
+        // Determine what text to show as a tooltip for this search_item.
+        if (SampleBin.hasItem(search_item)) {
+          return 'already added to analysis';
```

Comment:
Okay, I'm making the button and the tool tips capitalized (initial caps only, or sentence case). I'll go more in depth on my thinking about the ALL CAPS issue in the follow-up over at #142, but I don't have time to write that up at the moment.